src/styles/blocks/header.scss

```diff
@@ -0,0 +1,82 @@
+@import '../utils/variables';
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  box-sizing: border-box;
+
+  padding: 0 50px;
+
+  &-logo {
+    height: 40px;
+    width: 40px;
+  }
+}
```

Comment:
According to the BEM methodology, the element should be separated with two underscores from its block. For example, `.block__element`. In this case, `.header-logo` should be `.header__logo` to follow the BEM naming convention.